DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml

```diff
@@ -5,7 +5,7 @@
 {
 <p class="nhsuk-u-secondary-text-color nhsuk-u-margin-bottom-0">
   @Model.TimeSpentSummary spent
-  <span class="visually-hidden">on this tutorial</span>
+  <span class="nhsuk-u-visually-hidden">on this tutorial</span>

```

Comment:
How did this work if we were using the wrong class? Is `visually-hidden` our own class, i.e. one we've added css for? 🙂

Reply:
I asked @SChandarana and he doesn't really remember where he got it from, the `visually-hidden` class does exist in our CSS and is being used in some Learning Portal pages but we're using `nhsuk-u-visually-hidden` on all Learning Menu pages so I changed it for consistency.

Reply:
Okay, fair enough 🙂